processes-endpoint.md

```diff
@@ -140,32 +140,144 @@ In a future version, a solution supporting a live-feed can also be included.
 ## Execution File Output List
 **GET /api/system/processes/<:process-id>/files**
 
-This endpoint will let an administrator download an output file created by a process. If the file is found, it will be presented as a download. This endpoint will support "Range" HTTP headers so that downloads can be paused and resumed.
-
+This endpoint will return a list of files that are associated with the process, this can be files uploaded by the end user for imports or files generated by the script like for exports of data.
+The files are grouped by "type", the type can be anything but they refer to what the file represents, is a mapping file, an exported csv file, ...
```

Comment:
It would be quite some extra work to create an endpoint which specifies all types of all scripts (and also the API changes to be able to retrieve this information per scripts)
Just documenting it without an API would be something that's only useful to explain how it works (an example), but not an elaborate list of all types since each script can defined their own types and such documentation would tend to be always outdated